modules/bg_controller/appmesh-sf-task-token.py

```diff
@@ -0,0 +1,41 @@
+import json
```

Comment:
This file (appmesh-sf-task-token.py), and other files like iam.tf and lamba.tf are part of master already. And, apparently there are no changes to those files as part of this PR. You may remove these files from source branch

Reply:
if it appears in master how come it appears as a change ?